src/analyz/to_json.rs

```diff
@@ -1,3 +1,4 @@
+use rustc_abi::ExternAbi;
```

Comment:
I'm looking at a few errors in `to_json.rs` next

Reply:
Fixed everything in `to_json.rs`.  All of the remaining errors are in `ty_json.rs`.